test/runtests.jl

```diff
@@ -1,5 +1,10 @@
-using NbodySimulator, Test
+using NBodySimulator, StaticArrays, LinearAlgebra
+using Test
 
-@testset "Initial test" begin
-	@test sqrt(36) == 6
-end
\ No newline at end of file
+include("nbody_lennard_jones_test.jl")
```

Comment:
can probably rename these now

Reply:
Right, I'll remove `nbody_` part from all the files here.